src/pages/hackathon.tsx

```diff
@@ -1,8 +1,45 @@
-import React from "react";
-import HackathonInfoBar from "../components/hackathon/HackathonInfoBar";
+import React from 'react';
+import HackathonInfoBar from '../components/hackathon/HackathonInfoBar';
+import LaunchCard from '../components/hackathon/LaunchCard';
+import { ReactComponent as LogisticsIcon } from '../assets/logistics.svg';
```

Comment:
nit: could you put all these new SVGs into assets/Hackathon/ pls? I've been organizing the rest like that cause without that the folder will probably have 20+ assets atm. Plus I don't think any of these components will get reused.